plugins/feeds/public/et_open.py

```diff
@@ -0,0 +1,138 @@
+import datetime
+import logging
+from datetime import timedelta, timezone
+from io import StringIO
+
+from idstools import rule
+
+from core import taskmanager
+from core.config.config import yeti_config
+from core.schemas import entity, indicator, task
+
+
+class ETOpen(task.FeedTask):
+    __SOURCE = (
```

Comment:
Remove the double underscores throughout the module (including functions, etc.) a single underscore is enough to mark a function as "private"